spec/example_app/app/assets/stylesheets/administrate/customize.scss

```diff
@@ -0,0 +1,4 @@
+.custom-administrate-css-hidden {
```

Comment:
Yes, underscores indicate a partial in Sass. By nature of declaring that `customizations` be imported into the main Sass manifest file here in Administrate, it essentially already a partial. The underscore declares to Sass that this file should not be compiled to CSS of its own.